catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/security/AccessControlPreIngestPlugin.java

```diff
@@ -97,7 +98,7 @@ public CreateRequest process(CreateRequest request) throws StopProcessingExcepti
                   ? AccessControlUtil.getOwner(metacard)
                   : subjectIdentity.getUniqueIdentifier(ownerSubject);
```

Comment:
Will it ever be possible to execute the second part of the ternary? A couple lines before, you check to see if any metacard doesn't have an owner

Reply:
It seems the owner check is tied to a `guest` check as well. There are still cases where we could get to this line and the owner may not be set.